cms/templates/cms/toolbar/toolbar_javascript.html

```diff
@@ -115,7 +115,7 @@
         'id': '{{ request.toolbar.clipboard.pk|unlocalize }}',
         'url': '{% if request.toolbar.clipboard.pk %}{% cms_admin_url "cms_page_clear_placeholder" request.toolbar.clipboard.pk %}{% endif %}'
     },
-    'messages': '{% filter escapejs %}{% if messages %}{% for message in messages %}{{ message }}{% endfor %}{% endif %}{% endfilter %}',
+    'messages': '{% filter escapejs %}{% if messages %}{% for message in messages %}{{ message|escapejs }}{% endfor %}{% endif %}{% endfilter %}',
```

Comment:
You are right. Good catch!
When I discovered the error, I was working with the older version of CMS and didn't notice it was already fixed in latest branch.